Contents/index.css

```diff
@@ -0,0 +1,225 @@
+/* --------General------- */
+
+body {
+  background-color: white;
+  color: #1a2236;
+  margin: 0;
+  background-image: url(./Contents/Resources/image_geometry_2.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: top calc(100vh - 100px) right;
+}
+
+p {
+  font-family: Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 1.5rem;
+  letter-spacing: -0.01em;
+  text-align: justify;
+}
+
+h1 {
+  font-family: 'Times New Roman', Times, serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 50px;
+  line-height: 100%;
+}
+
+/* --------Header------- */
+
+header {
+  width: 100%;
+  height: 100px;
+  padding: 20px;
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: space-between;
+  background-image: url(./Contents/Resources/image_geometry_1.svg);
+  background-size: contain;
+  background-repeat: no-repeat;
+  box-sizing: border-box;
+}
+
+#hamburger-menu {
+  font-size: 2rem;
+  height: 40px;
+  width: 40px;
+  text-align: center;
+}
+
+#top-left-shapes {
+  height: 100px;
+  left: 0;
+}
+
+/* --------Sections------- */
+
+section {
+  border-color: black;
+  height: auto;
+  width: 100%;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  background-color: rgba(255, 255, 255, 0.4);
+}
+
+/* --------intro section------- */
+
+section#intro,
+section#about {
+  padding: 25px;
+}
+
+button.dark {
+  height: 8vh;
+  background-color: #1a2236;
+  color: white;
+  font-size: 1.5rem;
+  border: 1px solid white;
+  margin-bottom: 50px;
+}
+
+button.dark:hover {
+  background-color: white;
+  color: #1a2236;
+  border: 1px solid #1a2236;
+}
+
+button.dark:disabled {
+  background-color: gray;
+}
```

Comment:
- [ ] Template requires a different style for a pressed button (bolder text and thicker border). Please make sure it works properly.